contracts/6/forwarder/BiconomyForwarder.sol

```diff
@@ -27,9 +27,7 @@ contract BiconomyForwarder is ERC20ForwardRequestTypes, Ownable{
 
     string public constant EIP712_DOMAIN_TYPE = "EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)";
 
-    bytes32 public constant REQUEST_TYPEHASH = keccak256(bytes("ERC20ForwardRequest(address from,address to,address token,uint256 txGas,uint256 tokenGasPrice,uint256 batchId,uint256 batchNonce,uint256 deadline,bytes32 dataHash)"));
-
-    mapping(address => uint256) public highestBatchId;
+    bytes32 public constant REQUEST_TYPEHASH = keccak256(bytes("ERC20ForwardRequest(address from,address to,address token,uint256 txGas,uint256 tokenGasPrice,uint256 batchId,uint256 batchNonce,uint256 deadline,bytes data)"));
```

Comment:
This contract inherits Ownable which assigns owner address as msg.sender... instead make a own version of Ownable contract where owner address is passed as parameter and use that.... refer gastoken-master branch for such implementation.

Reply:
ok done for BiconomyForwarder and FeeManager contracts. Should I do it for all contracts that uses open zeppelin's ownable? ie OracleAggregator & ERC20Forwarder (both use modifiers onlyOwner)

Reply:
done for oracle aggregator also. for ERC20 forwarder should discuss today perhaps